### PR TITLE
styles added

### PR DESCRIPTION
**Pull request by katy1313**

### CODE REVIEW

File: js/index.js [hunk 1]
```diff
@@ -12,7 +104,65 @@ fetch("https://api.open-meteo.com/v1/forecast?latitude=52.52&longitude=13.41&hou
...
+//         console.log(forecast);
+//     } 
+//     //Displaying reports
+//     const today = document.querySelector('#today');
+//     const todayData = today.querySelector('p');
+//     todayData.innerHTML = forecast.current.temperature_2m;
+// })
+// .catch(error => {
+//     console.log(error);
+// })
```
Comment (Lee-Rice) [inline js/index.js]: I'd recommend removing this commented out code before merging.

File: js/index.js [hunk 2]
```diff
@@ -1,5 +1,97 @@
-// Creating fetch
-fetch("https://api.open-meteo.com/v1/forecast?latitude=52.52&longitude=13.41&hourly=temperature_2m&timezone=America%2FNew_York&models=gfs_seamless")
+//Switching between units
+
+let btns = document.querySelectorAll('button');
+for(btn of btns) {
+    if (btn.innerHTML === 'Cº') {
```
Comment (Lee-Rice) [inline js/index.js:5]: Seems like a great way to be sure you're selecting the correct button. Well done!

File: js/index.js [hunk 3]
```diff
@@ -1,5 +1,97 @@
...
+                    throw new Error("Not Found");
+                }
+                return response.json();
+            })
+            .then(report => {
+                if(report.length === 0) {
+                    throw new Error("No report exist");
+                } else {
+                    console.log(report);
+                }
```
Comment (Lee-Rice) [inline js/index.js:19]: Good use of error handling.

File: js/index.js [hunk 4]
```diff
@@ -12,7 +104,65 @@ fetch("https://api.open-meteo.com/v1/forecast?latitude=52.52&longitude=13.41&hou
     } else {
         console.log(report);
     } 
+    //Displaying reports
+    const city = document.querySelector('.city');
+    city.innerHTML = report.timezone;
+    const hourlyTemp = document.querySelector('.grid');
+    hourlyTemp.innerHTML = (report.hourly.temperature_2m).join(" ");
```
Comment (Lee-Rice) [inline js/index.js]: For readability, I would recommend having these listed along with the hour they pertain to. For example, something like
1:00 25.3 C
2:00 24.8 C
and so on.

File: js/index.js [hunk 5]
```diff
@@ -1,5 +1,97 @@
...
+                console.log(error);
+            })
+        })
+    }
+    
+}
+
+// Displaying city, default current temp and the hourly temperature
+
+fetch("https://api.open-meteo.com/v1/forecast?latitude=27.907&longitude=-82.6909&current=temperature_2m,relative_humidity_2m,is_day,precipitation&temperature_unit=fahrenheit&timezone=America%2FNew_York&forecast_days=1&models=gfs_seamless")
```
Comment (Lee-Rice) [inline js/index.js]: At this point, it doesn't look like you're using the relative humidity and precipitation. I'd recommend removing those from your API call unless you were planning on using them.

**Review by Lee-Rice — COMMENTED** [review]
This is looking great so far. Since this is a work in progress, you may already be planning to do the things I point out in the comments, but I thought it would be best to mention them, just in case.

I noticed you're using temperature for the three current data points. I think they're looking for a second data point. Something like precipitation, or snowfall. Since you're going to be pulling for different time frames, the way you're doing it could be just fine though. I'd recommend checking with one of the class leaders to be sure.

**Comment by katy1313** [comment]
> This is looking great so far. Since this is a work in progress, you may already be planning to do the things I point out in the comments, but I thought it would be best to mention them, just in case.
> 
> I noticed you're using temperature for the three current data points. I think they're looking for a second data point. Something like precipitation, or snowfall. Since you're going to be pulling for different time frames, the way you're doing it could be just fine though. I'd recommend checking with one of the class leaders to be sure.

I am pulling 3 data points in one fetch (humidity, precipitation and temperature), that I am going to display.

**Pull request merged**